starwars/src/components/StarWarsCard.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import StarWarsData from './StarWarsData';
+
+const StarWarsCard = props => {
+    return (
+        <div className= "card-container">
+            {props.starwarsChars.map(char => (
+            <StarWarsData
+                starwarsChars={char}
+                key= {char.name}
```

Comment:
Good job giving it a key! Keys don't have to be numbers, they just have to be unique, so using the character name fits this requirement.